src/core_plugins/kibana/common/field_formats/types/boolean.js

```diff
@@ -0,0 +1,29 @@
+import { asPrettyString } from '../../utils/as_pretty_string';
```

Comment:
The types changed a lot. Instead of being wrapped in a function, now they just export the FieldFormat class